src/api/plugins.js

```diff
@@ -15,7 +16,7 @@ const getAll = () => config.get('plugins') || [];
  * @return {Boolean}
  */
 // eslint-disable-next-line no-bitwise
-const isEnabled = plugin => ~getAll().indexOf(plugin);
+const isEnabled = plugin => ~getAllEnabled().indexOf(plugin);
```

Comment:
what is the type of the `plugin` here?
if it is a string, then it would be enough with `getAllEnabled().includes(plugin)`